diff --git a/node_modules/pdf-poppler/index.js b/node_modules/pdf-poppler/index.js
index a4a48b3..7cddb1c 100644
--- a/node_modules/pdf-poppler/index.js
+++ b/node_modules/pdf-poppler/index.js
@@ -2,8 +2,11 @@ const os = require('os');
 const path = require('path');
 const {spawn} = require('child_process');
 
+
+
 let platform = os.platform();
-if (!['darwin', 'win32'].includes(platform)) {
+
+if (!['darwin', 'win32', 'linux'].includes(platform)) {
     console.error(`${platform} is NOT supported.`);
     process.exit(1);
 }
@@ -28,7 +31,9 @@ if (platform === 'win32') {
     // for electron ASAR
     popplerPath = popplerPath.replace(".asar", ".asar.unpacked");
 }
-else if (platform === 'darwin') {
+else if (platform === 'linux') {
+    popplerPath = '/usr/bin'; // or `which pdftoppm` path
+}else if (platform === 'darwin') {
     popplerPath = path.join(
         __dirname,
         'lib',
